CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+-   Navigate into newly created folders and scroll to newly created passwords
```

Comment:
This should go under the `### Added` subheading.

Reply:
Done